packages/web-components/src/components/rux-segmented-button/test/segmented-button.spec.ts

```diff
@@ -113,6 +113,41 @@ test.describe('Segmented-button', () => {
         await expect(changeEvent).toHaveReceivedEventTimes(1)
         await expect(changeEvent).toHaveReceivedEventDetail('Second segment')
     })
+    test('it properly reselects a pre-selected segment', async ({ page }) => {
+        const template = `
```

Comment:
Because this one's a smol change, try commenting out the change you made. The test should fail. When I comment it out, the test still passes. Can we think of a different approach here to capture the issue we're seeing?

Reply:
Playwright not working how I thought it was. This should be solved :)
I added some evaluated to grab the properties and then added expect(evaluation).toBe(true) and it was happy. with that.